packages/cli/src/cmds/stats/directory/statsForDirectory.ts

```diff
@@ -0,0 +1,303 @@
+import { chdir } from 'process';
+import * as fs from 'fs';
+import UI from '../../userInteraction';
+import Telemetry from '../../../telemetry';
+import { listAppMapFiles, verbose } from '../../../utils';
+import { Event, buildAppMap } from '@appland/models';
+import { AppMapSizeTable, SortedAppMapSize } from '../types/appMapSize';
+import { FunctionExecutionTime, SlowestExecutionTime } from '../types/functionExecutionTime';
+import chalk from 'chalk';
+import { relative } from 'path';
+
+const MINIMUM_APPMAP_SIZE = (1024 * 1024) / 1;
+
+export function sizeInMB(size: number): number {
```

Comment:
Is this all unchanged from the existing implementation?

Reply:
Yeah, this is unchanged